.gitlab/choco_deploy.yml

```diff
@@ -20,5 +20,10 @@ publish_choco_7_x64:
     - mkdir nupkg
     - copy omnibus\pkg\*.nupkg nupkg\
     - Get-ChildItem nupkg
-    - docker run --rm -v "$(Get-Location):c:\mnt" -e CHOCOLATEY_API_KEY=${chocolateyApiKey} 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/windows_1809_${ARCH}:$Env:DATADOG_AGENT_WINBUILDIMAGES c:\mnt\tasks\winbuildscripts\chocopush.bat
+    - >
```

Comment:
What - > means here? Multiline?

Reply:
Yes ! This SO post explains it really well: https://stackoverflow.com/a/21699210